recipe/meta.yaml

```diff
@@ -31,7 +31,7 @@ requirements:
     - opt_einsum
     - scipy >=1.11.1
     - ml_dtypes >=0.4.0
-    - jaxlib >={{ minimum_jaxlib_version }},<={{ version }}
+    - jaxlib >=0.5.1,<=0.5.2
```

Comment:
@beckermr, is it intentional that the bot is replacing jinja variables within a recipe now? That seems unintentional...

Reply:
This is technically not a grave concern, can just remove the jinja variables since it seems that pip tarball is being parsed correctly.

Reply:
It could be fine to remove `set minimum_jaxlib_version` and move the comment right above that to the requirements section.

Reply:
No this is a bug. Please leave this pr open so I can investigate.